### ISSUE TITLE
abouttraining loss

### ISSUE THREAD

**Issue by Godaddy-xie** [Wolfwjs/GANet#23]
Can u provide the training log? i have try toI tried to fit on a small dataset (1000 culanes, or my own dataset , batchsize = 1), but focalloss1 has been around 1.35 (culane 800*320) and cannot be decreased.
i have let other loss weight zero.
![image](https://user-images.githubusercontent.com/54409021/178033155-01adaffc-ce53-4565-ab63-b831f1940177.png)
 
![image](https://user-images.githubusercontent.com/54409021/178033029-66d70001-27cc-4b03-9d18-7a94cebc9fe7.png)

**Comment by spyflying** [comment]
It is unusual that the focalloss keeps high. I suppose batchsize=1 might be the reason. You can try to use larger batchsize during training. Also, try to use the provided checkpoint for parameter initialization and finetune on your dataset with smaller learing rate. Since the original log is not available now, I trained on tusimple for several epochs and paste the log below.  Hope it could help you.

![focal](https://user-images.githubusercontent.com/22930712/187151311-e6f60bfd-f036-4b22-a282-4e23d7acc43e.png)

<img width="1342" alt="log" src="https://user-images.githubusercontent.com/22930712/187151108-c3400ec0-9f54-41e1-a70d-34600fb43a48.png">
